velox/exec/StreamingWindowBuild.h

```diff
@@ -28,10 +28,20 @@ class StreamingWindowBuild : public WindowBuild {
  public:
   StreamingWindowBuild(
       const std::shared_ptr<const core::WindowNode>& windowNode,
-      velox::memory::MemoryPool* pool);
+      velox::memory::MemoryPool* pool,
+      const common::SpillConfig* spillConfig,
+      tsan_atomic<bool>* nonReclaimableSection);
 
   void addInput(RowVectorPtr input) override;
 
+  void spill() override {
+    // Spilling is not supported.
```

Comment:
Please add VELOX_NYI.

Reply:
VELOX_NYI() would not be appropriate. If this operator cannot spill, maybe some other operator can. No need to fail the query.

Reply:
Great. Thanks !